templates/varnish.j2

```diff
@@ -97,7 +91,8 @@ VARNISH_TTL=120
 #
 # # DAEMON_OPTS is used by the init script.  If you add or remove options, make
 # # sure you update this section, too.
-DAEMON_OPTS="-a ${VARNISH_LISTEN_ADDRESS}:${VARNISH_LISTEN_PORT} \
+DAEMON_OPTS="{% for p in varnish_listen_ports %}-a :{{ p.port }}{% if p.proxy_proto is defined and p.proxy_proto and varnish_version|float >= 4.1 %},PROXY{% endif %} \
```

Comment:
By the way, a future improvement could be to actually define the listening IP address. Currently, Varnish has to listen on all local interfaces, but it may not be a good idea in some cases. So the `varnish_listen_ports.port` could be augmented by a `varnish_listen_ports.address` variable.